app/logic/events.py

```diff
@@ -318,3 +319,10 @@ def preprocessEventData(eventData):
         eventData['facilitators'] = list(User.select().join(EventFacilitator).where(EventFacilitator.event == event))
 
     return eventData
+
+
+def getTomorrowsEvents():
+    """Grabs each event that occurs tomorrow"""
+    tomorrowDate = date.today() + timedelta(days=1)
+    events = list(Event.select().where(Event.startDate==tomorrowDate))
```

Comment:
See if you can change list() to the [ ] notation. It actually calculates faster and is easier to read.

Reply:
_Lame_